### ISSUE TITLE
Remove `mc:` debug messages from client-facing logs

### ISSUE THREAD

**Issue by technosophos** [deis/builder#56]
During a `git push deis`, the following debug messages are displayed.

```
mc: Configuration written to [/var/minio-conf/config.json]. Please update your access credentials.
mc: Successfully created ‘/var/minio-conf/share’.
mc: Initialized share uploads ‘/var/minio-conf/share/uploads.json’ file.
mc: Initialized share downloads ‘/var/minio-conf/share/downloads.json’ file.
```

These are confusing to the user. Can we write them to the log instead?

**Comment by arschles** [comment]
cc/ @gabrtv - marked as a showstopper bug in alpha

**Comment by gabrtv** [comment]
This specific message was fixed via #60, so closing this out. There are a few other related issues: deis/slugbuilder#16 and deis/slugrunner#7.
